src/swap/defi/uni-v2-based/plugins/tombSwap.js

```diff
@@ -0,0 +1,153 @@
+// @flow
+
+import {
+  type EdgeCorePluginOptions,
+  type EdgeSpendInfo,
+  type EdgeSwapInfo,
+  type EdgeSwapPlugin,
+  type EdgeSwapQuote,
+  type EdgeSwapRequest,
+  type EdgeTransaction
+} from 'edge-core-js/types'
+import { ethers } from 'ethers'
+
+import { getInOutTokenAddresses } from '../../defiUtils.js'
+import { tombSwapRouter } from '../uniV2Contracts.js'
+import {
+  getSwapAmounts,
+  getSwapTransactions,
+  makeUniV2EdgeSwapQuote
+} from '../uniV2Utils.js'
+
+const EXPIRATION_MS = 1000 * 20 * 60
+const SLIPPAGE = '0.05'
+
+const swapInfo: EdgeSwapInfo = {
+  pluginId: 'tombSwap',
+  displayName: 'TombSwap',
+  supportEmail: '',
+  supportUrl: 'https://discord.gg/vANnESmVdz'
+}
+
+export function makeTombSwapPlugin(
```

Comment:
If the implementation is nearly the same as `makeSpookySwapPlugin` then why not `makeUniswapPlugin` with parameters in the plugin info which determine the contract address to use, etc?

Reply:
I'll secede this request for a later refactor.